src/generateDapp.ts

```diff
@@ -138,6 +146,47 @@ export async function generateDapp(selection: Selections) {
       case "boilerplate-template":
         await generateEnvFile();
         break;
+      case "clicker-game-tg-mini-app-template":
+        const dir = "frontend/components";
+
+        if (selection.signingOption === "explicit") {
+          await generateEnvFile();
+          await rename(dir, "ExplicitSigningCounter.tsx", "Counter.tsx");
```

Comment:
might be easier to put all `ExplicitSigning` files and `SeamlessSigning` files in two separate directories, and then based on the selection just copy the directory content to the `components` folder.

```
| explicitSigning
-- Counter.tsx
-- WalletProvider.tsx
-- WalletSelector.tsx

| seamlessSigning
-- Counter.tsx
-- WalletProvider.tsx
-- WalletSelector.tsx
```